changeset:   7754:f38cf80e1163
user:        Aidan Kehoe <kehoea@parhasard.net>
date:        Fri Aug 29 07:35:13 2025 +0100
summary:     Fall back to INCBIN(), avoid #embed with GCC 15. Thank you Leonid Kopylov!

diff -r 887fd62b88bd -r f38cf80e1163 src/dump-data.c
--- a/src/dump-data.c	Tue Aug 26 18:27:00 2025 +0100
+++ b/src/dump-data.c	Fri Aug 29 07:35:13 2025 +0100
@@ -24,7 +24,7 @@
 #include "lisp.h"
 #include "dump-data.h"
 
-#ifdef __has_embed
+#if defined (__has_embed) && !(GCC_VERSION > NEED_GCC (15, 0, 0))
 #include <stdalign.h>
 
 alignas (16) static Rawbyte dumped_data[] = {

